### ISSUE TITLE
Session set's itself to previous value on refresh

### ISSUE THREAD

**Issue by BradRyan** [cramrov/meteor-session-amplify#3]
Here is the scenario.  I'm using SessionAmplify in a wizard to store values locally.  At the end of each step, I create an object with the field values for that step.  If at some point the user wants to reset the form, I have a button on the bottom of the wizard to "start over".  When pressed, it sets the SessionAmplify objects for each step to undefined.  I log the event and it verifies the object is empty.  

When I return to step 1, everything is working as it should (the fields have helpers to grab values from the SessionAmplify object if it exists) UNTIL I refresh the page.  Then, for some reason that I cannot explain, the SessionAmplify set's itself to the previous object that was stored, causing all of the old values to appear in the form (before setting the SessionAmplify object to undefined).  Any ideas on what's going on here?

**Comment by BradRyan** [comment]
Just an update to this... It seems like SessionAmplify is setting itself to it's previous value on page reload in the event that it is undefined and had a previous value.  When I set my SessionAmplify object to something else (0 in my test), I did not have the issue on page reload.

``` javascript
if (SessionAmplify.get("mysession") == undefined) {
   SessionAmplify.set("mysession", previousValue)
}
```

**Comment by seeekr** [comment]
Yeah, this kinda makes SessionAmplify a lot less useful than it could be. Or at least that much harder to use when you have to do manually manage what values mean "not set" even though in JS terms they're actually not `undefined` or `null`.
I understand that the problem lies in the underlying amplify.js library being used. I think right now the best approach is to avoid amplify.js and therefore SessionAmplify and use something else, if you need to be able to unset values without it being a hassle. (I'm using something localstorage-based right now that more directly supports unsetting.)
